compiling_everything_linux.md

```diff
@@ -0,0 +1,65 @@
+# Compiling Teeworlds on Linux
+
+**IMPORTANT NOTE:** SDL and Freetype libs are not shipped with Teeworlds 0.7.x. You must download them separately.
+
+**Q: What is BAM?**
+
+Bam is the [build system made by matricks](http://matricks.github.io/bam/) used in Teeworlds.
+
+#### Setup
+1. Use your package manager (apt-get, emerge or whatever is used on your distribution) to install the following (you will need the header files):
+    - gcc
+    - g++
+    - python
+    - alsa (asound)
+    - gl
+    - glu
+    - x11
+    - libsdl
+    - freetype
+2. Download and unzip [Teeworlds-source](https://github.com/teeworlds/teeworlds/releases) or [Teeworlds-latest-source](https://github.com/teeworlds/teeworlds)
```

Comment:
I would say either use English (Teeworlds source, Teeworlds latest source) or keep the nomenclature from the downloads (teeworlds-version, etc).